### PR TITLE
fixes anchor links in chrome on pages with tabs

### PR DESCRIPTION
**Pull request by fitzage**
### What does this PR do? What is the motivation?
https://datadoghq.atlassian.net/browse/WEB-4932

Fixes weird table of contents behavior reported in the above ticket.

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

1. Visit a page with tabs and a table of contents, like this one: https://docs-staging.datadoghq.com/fitzage/anchor-fix/real_user_monitoring/browser/advanced_configuration/?tab=npm
2. Click on one of the table of contents links.
3. Click the same table of contents link again. It should work correctly instead of jumping to the top of the page.

### CODE REVIEW

**Review by drichards-87 — APPROVED** [review]
@fitzage. Looks good to me. Are you waiting for someone from Websites to review before we merge?

**Review by LisianeTu — APPROVED** [review]

**Comment by fitzage** [comment]
@drichards-87 Yep, but looks like we just got it. Merge away!

**Pull request merged**
